CHANGES.md

```diff
@@ -5,6 +5,9 @@ Airbase 148
 * Dependency updates:
   - Jackson 2.15.3 (from 2.15.2)
 
+* Plugin updates:
+  - errorprone 2.22.0 (from 2.21.1)
```

Comment:
This is actually a dependency update. We don't (yet) have an Error Prone plugin configuration in Airbase.

Reply:
I'll correct changelog